fimsauthz-api/nup-roles.yaml

```diff
@@ -6,106 +6,55 @@ UTM_AUTHORITY:
-        This role is typically associated with a User whose authorization is out of band WRT FIMS oauth.
-    authority: UTM_AUTHORITY
-    examples:
-        - "An IT admin manages FIMS-AZ logfiles."
-        - "An IT admin reads the master USS list to update FIMS-AZ's data table containing the USSs, their scopes and their credentials."
-    scopes:
-        - utm.nasa.gov_read.fimsadmin
-
-FIMS_SME:
-    description: |
-        A role is assigned to user who is a FIMS Subject Matter Expert (SME) who
-        manages airspace constraints and other aeronautical data
-        This user could be a client to the FIMS Ops server using a User Interface (rather than CLI).
-        This role is typically associated with a user whose authorization is out of band WRT FIMS oauth.
-    authority: ANSP
-    examples:
-        - "An airspace Subject Matter Expert composes an airspace constraint and submits this constraint to FIMS_OPS."
-
-FIMS_AUTHZ:
-    description: >-
-        This is the service role supporting the FIMS Authorization server.
-        This role cannot create, modify or delete UTM Identity data, rather access is read-only.
-    authority: UTM_AUTHORITY
-    examples:
-        - "When FIMS AZ services the token endpoint, it reads a data table containing USS identities and their scopes."
-
 USS_BASIC:
-    description: A USS. The UTM Authority assigns role after
-        out-of-band vetting. Vetting includes some testing mechanism and liability assumptions TBD.
+    description: |
+        A USS. The UTM Authority assigns role after out-of-band vetting. 
+        Vetting includes some testing mechanism and liability assumptions TBD.
 
         According to the ASTM Spec - 1.0
-           - Client may perform planning, strategic conflict detection, conformance monitoring activities
-                (including reading information about Constraints), and set the availability of USS in DSS
+           - Client may perform actions encompassed by the strategic coordination role including strategic conflict detection.
+           - Client may read constraint references from the DSS and details from the partner USSs according to the constraint processing role.
+           - Client may perform actions encompassed by the conformance monitoring for situational awareness role.
 
-           - Client may read Constraints from the DSS (references) and partner USSs (details).
     authority: UTM_AUTHORITY
     examples:
         - "USS queries Discovery service to discover other USSs in its 4D area."
     scopes:
         - utm.strategic_coordination
-        - utm.constraint_management
         - utm.constraint_processing
         - utm.conformance_monitoring_sa
-        - utm.availability_arbitration
 
-USS_CONSTRAINT_MANAGER:
+CONSTRAINT_MANAGER:
     description: |
-        A USS with this role can publish airspace constraints such as UASVolumeReservations (UVRs).
+        An user with this role can publish airspace constraints.
```

Comment:
a user or service (or simply "This role can..."

Reply:
fixed